src/ert/libres_facade.py

```diff
@@ -505,14 +498,15 @@ def gen_kw_keys(self) -> List[str]:
 
         gen_kw_list = []
         for key in gen_kw_keys:
-            enkf_config_node = self._enkf_main.ensembleConfig().getNode(key)
-            gen_kw_config = enkf_config_node.getModelConfig()
+            gen_kw_config = self._enkf_main.ensembleConfig().get_keyword_model_config(
+                key
+            )
             assert isinstance(gen_kw_config, GenKwConfig)
 
-            for keyword_index, keyword in enumerate(gen_kw_config):
+            for keyword in gen_kw_config:
                 gen_kw_list.append(f"{key}:{keyword}")
 
-                if gen_kw_config.shouldUseLogScale(keyword_index):
+                if gen_kw_config.shouldUseLogScale(keyword):
                     gen_kw_list.append(f"LOG10_{key}:{keyword}")
```

Comment:
This is never tested with `gen_kw_config.shouldUseLogScale(keyword) == True`

Reply:
Added.